README.md

```diff
@@ -15,6 +16,12 @@ a Geographic Information System used for geospatial data management and
 analysis, image processing, graphics/map production, spatial modeling, and
 visualization.
 
+## Jupyter Notebook
+Launch this repository in binder and experiment with GRASS's Python API in Jupyter Notebooks by clicking the button below:
```

Comment:
binder -> Binder: I think in text they use Binder rather than binder which is in logos.